### PR TITLE
chore: upgrade sortedoracles

### LINKED ISSUE #96: Update MU03 deployment scripts and CGP to upgrade the SortedOracles implementation
**Acceptance Criteria**

- [ ] write the deploy script 
- [ ] update the SortedOracles implementation in the gov script

### PR DESCRIPTION
**Pull request by ninocomputer**
### Description

- [ ] Updated deployment script to include SortedOracles
- [ ] Extended MU03checks 
- [ ] Deployed on alfajores/baklava

### Other changes

n/a

### Tested

governance simulation works

### Related issues

- Fixes #96

### Backwards compatibility

n/a

### Documentation

n/a

### CODE REVIEW

File: script/upgrades/MU03/MU03Checks.sol [hunk 1]
```diff
@@ -17,6 +17,7 @@ import { IBroker } from "mento-core-2.2.0/interfaces/IBroker.sol";
 import { IERC20Metadata } from "mento-core-2.2.0/common/interfaces/IERC20Metadata.sol";
 
 import { BiPoolManagerProxy } from "mento-core-2.2.0/proxies/BiPoolManagerProxy.sol";
+import { SortedOraclesProxy } from "mento-core-2.0.0/proxies/SortedOraclesProxy.sol";
```
Comment (bowd) [inline script/upgrades/MU03/MU03Checks.sol:20]: Let's use the `2.2.0` consistently in `MU03`, even tho in practice this ends up being the same contract in this case, if we mix up different versions we might introduce subtle bugs.

File: script/upgrades/MU03/MU03Checks.sol [hunk 2]
```diff
@@ -134,6 +142,21 @@ contract MU03Checks is Script, Test {
     console2.log("\tBiPoolManagerProxy has a correct implementation address 🫡");
   }
 
+  function verifySortedOracles() internal view {
+    SortedOraclesProxy soProxy = SortedOraclesProxy(sortedOraclesProxy);
```
Comment (bowd) [inline script/upgrades/MU03/MU03Checks.sol:146]: This is correct but I kinda want us to move away from using named proxies so for the sake of that I'd import the base `Proxy` contract instead and cast it as that here, just to future-proof the system.

**Review by bowd — CHANGES_REQUESTED** [review]
Small nits ⛩️

File: script/upgrades/MU03/MU03Checks.sol (hunk 1 above)
Comment (ninocomputer) [reply script/upgrades/MU03/MU03Checks.sol:20]: good catch! it was the co-pilot 😤

**Review by bowd — APPROVED** [review]

**Review by philbow61 — APPROVED** [review]

**Pull request merged**